src/_locales/fr/messages.json

```diff
@@ -231,7 +231,7 @@
     "hash": "5f54e42067e19895a2641b17a5233418"
   },
   "ariaMessageSuccess": {
-    "message": "message de succès",
+    "message": "message de validation",
```

Comment:
I know no French, but this string is actually used for all success messages, so I am unsure whether validsation is correct here.

Reply:
The only one I can think of at the top of my head is the one, when you reset the settings, it shows a success message including a button to revert the reset:

![grafik](https://github.com/rugk/mastodon-simplified-federation/assets/11966684/2b018ebd-114e-4a9d-93c7-a6b692a5bad4)

(Sorry German screenshot)